Ports/.port_include.sh

```diff
@@ -28,7 +28,7 @@ enable_ccache() {
     if [ "${USE_CCACHE:-true}" = "true" ] && command -v ccache &>/dev/null; then
         ccache_tooldir="${SERENITY_BUILD_DIR}/ccache"
         mkdir -p "$ccache_tooldir"
-        for tool in cc clang gcc c++ clang++ g++; do
+        for tool in clang gcc clang++ g++; do
```

Comment:
The `c++` binary actually exists (at least on `i686 GNU`), so we should probably keep that link to not accidentally skip `ccache` if anything uses it.

Reply:
You are right, I have readded creation of the `i686-pc-serenity-c++` ccache link.

Reply:
Please squash the fixup change into the first commit, and update the commit/PR title while at it (as it still mentions `c++` being removed).

Afterwards you can mark this comment as "resolved", so it looks more ready. :^)